### DISCUSSION TITLE
Is there a way to format text with handlebars?

### DISCUSSION THREAD

**Discussion by tw02d008** [Budibase/budibase#10025] [Help] [answered]
Sorry if this is a dumb question, I am new to budibase and handlebars and I am looking for a way to format text depended on a value.
I want to write some text bold related to a database value
``` 
{{#if (gte Repeater.TaskStatus_Overview.daysleft 10)}}
       {{ Repeater.TaskStatus_Overview.daysleft }} Tage übrig
{{else}}
       <b> {{ Repeater.TaskStatus_Overview.daysleft }} Tage übrig</b>
{{/if}}
``` 

unfortunately the HTML Tags are ignored and not parsed and just displayed as text. Is there a way to achieve this?

**Comment by andz-bb** [comment, marked as answer]
Hey @tw02d008

I think your Handlebars expression should work properly within an Embed component as that will render whatever HTML it returns.

If you want to use another component such as a Paragraph the easiest way to conditionally format it is to use the component's Conditions.

Here's a quick example:
![Screenshot 2023-03-14 at 19 20 20](https://user-images.githubusercontent.com/110921612/225114378-d385a4d9-0732-41fe-9f9f-fda4c00658a4.png)

For the text returned in the paragraph component you can return `{{ Repeater.TaskStatus_Overview.daysleft }} Tage übrig` in all cases. Then in the paragraph component's conditions you can add one that updates the component's Bold setting to true in cases where the daysLeft binding is less than or equal to 10. You can access the conditions settings at the bottom of the right side bar for the component. 

Hope this helps! If you have any questions feel free to ask here or [check out our docs page on Conditional UI here.](https://docs.budibase.com/docs/conditions) Have a good evening 😊

**Reply by tw02d008** [reply]
Great. Thanks for your help
